app/src/org/commcare/models/database/SqlStorage.java

```diff
@@ -81,6 +81,14 @@ public SqlStorage(String table, Class<? extends T> ctype, AndroidDbHelper helper
         }
     }
 
+
+    @Override
+    public Vector<Integer> getAllIDs() {
```

Comment:
@shubham1g5 Why did this make it into the PR? Testing?  It doesn't seem to be referenced by the actual callout, is the underlying implementation functional but less performant?

Reply:
nevermind, just saw the other Pr. Following up there.